src/include/duckdb/storage/serialization/parsed_expression.json

```diff
@@ -0,0 +1,376 @@
+[
```

Comment:
Not sure exactly how, but I would try to enforce (probably via script run in CI) that only 'good' changes are done to this file, where by good I mean that members should not be removed / shuffled / renamed (= doing so triggers incompatibility).

Or via a script comparing two of these json and returning either "compatible" or not. That way we could use say json from 1.0 and json from 1.x, and check whether they are (forward!) compatible.
This partially could be reached by testing, but could be nice to have it also at the json level.

Reply:
Yes that is a good idea as well